integration_tests/conftest.py

```diff
@@ -47,7 +47,7 @@ def geth(tmp_path_factory):
     yield from setup_geth(path, 8545)
 
 
-@pytest.fixture(scope="session", params=["cronos", "geth"])
+@pytest.fixture(scope="session", params=["cronos", "geth", "cronos-ws"])
```

Comment:
all the test cases which use the `cluster` fixture will be run an extra time against Cronos WebSocket endpoint.